test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java

```diff
@@ -235,6 +238,7 @@ private static void showSuites(String[] suites) {
     }
 
     public static void main(String[] args) throws Exception {
+        SecurityUtils.removeFromDisabledTlsAlgs("TLS_RSA_*");
```

Comment:
This test verifies that only the enabled suites are what are expected, and we didn't accidentally re-enable a cipher suite. You should not be re-enabling any disabled suites. Instead remove TLS_RSA from the static String arrays.

Reply:
Done. I think that's what I did in my other branch, but then decided to just disable TLS_RSA everywhere:
https://github.com/openjdk/jdk/pull/21911/files#diff-e60d5c4686491da1ba1ccd456053fcec9cffcaae304546235066d39b37fd96e0